packages/codemirror-graphql/src/lint.js

```diff
@@ -33,7 +33,8 @@ const TYPE = {
  */
 CodeMirror.registerHelper('lint', 'graphql', (text, options) => {
   const schema = options.schema;
-  const rawResults = getDiagnostics(text, schema);
+  const validationsRules = options.validationsRules;
+  const rawResults = getDiagnostics(text, schema, validationsRules);
```

Comment:
@acao I think I am passing to to `getDiagnostics` here?

Reply:
ope, there ya go. hard reading PRs on my phone. I’ll check this out on my laptop and add a few commits soon!

Reply:
Aight! Would love to see this published as soon as possible 😇

Reply:
there are some issues with flow + codemirror-graphql that are blocking publish, as you can see in the open PR. will work on those soon. I have several workshops I'm putting on this weekend, so it may have to wait til next weekend, sorry!

Reply:
No worries in the meantime I can still apply it as a apathy :) Let me know if I can help you on something else in the meantime

Reply:
just a quick commit to add it to monaco-graphql would be great. the tricky thing would be figuring out how to load the validation rules over the webworker runtime boundary... possibly this makes sense as another follow-up PR, but since we *must* move to monaco-graphql so that our SR users can actually use GraphiQL, we need to be focusing more on that (which is why i've spent most of my time at gatsby working on it). So any feature we add to codemirror-graphql should be added to monaco-graphql as well ASAP, although we won't be backporting most of the new monaco-graphql features back to codemirror-graphql, such as SDL support, peek definitions, or prettier formatting

Reply:
I will try some of that tomorrow in a new branch

Reply:
great! i appreciate it. would you be able to add some tests for this PR?  for GraphiQL component itself, or you could also add some to `codemirror-graphql` if you like, if you don't mind writing mocha/chai/sinon tests for the latter?

Reply:
I will post-pone this for now.